### DISCUSSION TITLE
Mimir using k8s kustomize

### DISCUSSION THREAD

**Discussion by sahu-sunil** [grafana/mimir#4974] [Help and support]
Hi all, 

I want to deploy Mimir using K8s kustomization. Is there any reference or help document for this?

I converted helm templates to kustomize (using `helm template`) and then deployed but looks like Mimir components not working end to end properly.

**Comment by Logiraptor** [comment]
There's no reference for that method of deployment, but we may be able to help if you post more details about the issues you are seeing.
